example/src/Room.tsx

```diff
@@ -1,12 +1,12 @@
 import { Icon } from '@components/Icon';
 import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import { StyleSheet, Text, View, Pressable } from 'react-native';
 
 import { Controls } from './Controls';
 import { Settings } from './Settings';
 
-export const Room = ({ disconnect }: { disconnect: () => void }) => {
+export const Room = ({ navigation }) => {
```

Comment:
The back button should disconnect the call or there shouldn't be any back button. Now you can go back and connect again and that leads to weird behaviour. I can see that according to the designs there shouldn't be even a header. You can change that in navigator options https://reactnavigation.org/docs/native-stack-navigator#options

Reply:
Done.